### ISSUE TITLE
Must always pass quickReplies now in version 1.1.10

### ISSUE THREAD

**Issue by primemb** [fayeed/dash_chat#104]
I upgrade package to 1.1.10 and now i must always pass `quickReplies` param in `ChatMessage` or i get this error :

```
The getter 'values' was called on null.
Receiver: null
Tried calling: values
```

**Comment by m-j-g** [comment]
Came to report the same issue.

**Comment by fayeed** [comment]
@m-j-g This has been fixed in `1.1.11` just be live in a couple of minutes.

**Comment by tvanhuu** [comment]
> @m-j-g This has been fixed in `1.1.11` just be live in a couple of minutes.

I got the same error this afternoon. Thanks for fixing it <3
